websites/B/Bungie/presence.ts

```diff
@@ -0,0 +1,303 @@
+    presenceData.startTimestamp = browsingStamp;
+    presenceData.details = ("Readig Destiny Comics")
+  }
+    //else if (path.includes("/en/News/") && title) {
+  //presenceData.startTimestamp = browsingStamp;
+  //presenceData.details = ("Looking at " + title.textContent);
+  //}
+//Destiny Two NewLight don't know why I'm making this comment... I mean it's not like it's in the URL or anything like that...
+if  (path == "/7/en/Destiny/NewLight"){
+  presenceData.startTimestamp = browsingStamp;
+  presenceData.details = ("Looking at Destiny Two NewLight")
+}
+//Destiny Two Downloadable Content URLs
+  else  if (path == "/7/en/Destiny/Shadowkeep"){
+presenceData.startTimestamp = browsingStamp;
+presenceData.details = ("Checking out Destiny Two Shadowkeep")
+}
+else if (path =="/7/en/Destiny/Forsaken"){
+  presenceData.startTimestamp = browsingStamp;
+  presenceData.details = ("Checking out Destiny Two Forsaken")
+  }
+  // Destiny Two Seasons
+  else if (path =="/7/en/Seasons/SeasonOfTheWorthy"){
+    presenceData.startTimestamp = browsingStamp;
+    presenceData.details = ("Checking out Destiny Two Season of the Worthy")
+  }
+  else if (path =="/7/en/Seasons/SeasonOfDawn"){
+  presenceData.startTimestamp = browsingStamp;
+  presenceData.details = ("Checking out Destiny Two Season of Dawn")
+  }
+  else if (path =="/7/en/Seasons/SeasonOfTheUndying"){
+  presenceData.startTimestamp = browsingStamp; 
+  presenceData.details = ("Checking out Destiny Two Season of the Undying")
+  }
+//Bungie.net News Pages
+  else if (path.includes ("/en/Explore/Detail/News/")){
+  title = document.querySelector("#article-container > h1").innerHTML
+  presenceData.startTimestamp = browsingStamp;
+  presenceData.details = ("Reading "+ title)
+}
+
+
+else if (path =="/en/News"){
+presenceData.startTimestamp = browsingStamp; 
+presenceData.details = ("Cheking out the news from Bungie")
+}
+
+//Bungie,net Community Button (You can hover over it and it gives you a bunch of different options.)
+else if (path == "/en/ClanV2/Chat") {
+  titleSix = document.querySelector("#clanSideBar > div.container-left.customScroll.customScrollOff > a > div.compact-clanidentity-containter > div.clanNameContainer > h2").innerHTML
+  presenceData.startTimestamp = browsingStamp;
+  presenceData.details = ("Looking at thier clan " + titleSix.replace("<span>","").replace("</span>",""));
+}
+//Bungie.net Forum pages
+else if (path.includes ("/en/Forums/Topics")){
+  titleThirteen = document.querySelector("head > title")
+  presenceData.startTimestamp = browsingStamp;
+  presenceData.details = ("Looking at "+titleThirteen.innerHTML);
+}
+else if (path.includes ("/en/Forums/Post/")){
+  titleFourteen = document.querySelector("#topicPost > div > div.threadMeta > div > div > div.authorMeta > a")
+  titleTwo = document.querySelector("#topicPost > div > div.threadMeta > div > h1")
+  //document.querySelector("#topicPost > div > div.threadMeta > div > h1")
```

Comment:
why the comment?

Reply:
Different JavaScript path. Just kept it there just in case it worked out.

Reply:
it is 100% the same tho xD

Reply:
True. Sorry, I removed it now, but you just cannot see it yet.